src/FluentHub/App.xaml.cs

```diff
@@ -88,8 +81,24 @@ public App()
         /// </summary>
         private static IServiceProvider ConfigureServices()
         {
+            string logFilePath = Path.Combine(ApplicationData.Current.LocalFolder.Path, "logs", "Log.log");
+            var logger = new LoggerConfiguration()
```

Comment:
why did you create new log file folder?
cannot write same file?

Reply:
Can use the same file, but it's better to use .log extension instead .txt

Reply:
oh, ok. thanks.